### ISSUE TITLE
Fill-in-the-blank in long text cannot be focused

### ISSUE THREAD

**Issue by skyerGalaxy** [ankidroid/Anki-Android#17625]
I pasted a long text in a card and dug up a lot of holes

when I focus a hole in the center or bottom of text,
![Screenshot_20241218_230645.jpg](https://github.com/user-attachments/assets/e9f2e435-f043-4d7b-be29-41d6093e1b00)

when i tap show answer,card back the beginning of the article
![Screenshot_20241218_230719.jpg](https://github.com/user-attachments/assets/f042aaaa-0cc5-4d46-81c6-befbe7876216)

when begin next hole，card back the beginning of the article  again

Looking forward to your reply.

anki version  2.20.0

**Comment by david-allison** [comment]
Hi, could you try our new reviewer:

* Go to Settings -> About
* Tap the Anki logo 7 times
* Agree to enable the developer mode
* Open 'Developer Options'
* Enable 'New reviewer'

**Comment by caibimiaokong** [comment]
I opend it,but problem still exist

**Comment by BrayanDSO** [comment]
So, the issue is that you scroll down to some position of the text, but the screen is scrolled up to the top after you show the answer?

**Comment by caibimiaokong** [comment]
Yes, all in all, it's an autofocus problem for long text fill-in-the-blanks。
Thank you for your  efforts
